parsers/AR.py

```diff
@@ -101,14 +125,50 @@ def renewables_production_mix(zone_key: str, session: Session) -> Dict[str, dict
         }
         for production_info in sorted_production_list
     }
+    return renewables_production
+
+
+def regional_renewables_production_mix(
```

Comment:
So looking at the function, most parts could be integrated in the `renewables_production_mix` function, therefore limiting the amount of similar code.

Reply:
I combined the two into one function - please take a look again and let me know if it's acceptable!